### PR TITLE
Update DO how-to v0.18.1

### PR DESCRIPTION
**Pull request by eskombro**
~DO NOT MERGE UNTIL THE OFFICIAL DIGITALOCEAN IMAGE IS PUBLISHED (asked me if doubts)~

Changes to the DigitalOcean how-to

- Use `meilisearch` user to connect, instead of root
- Add health check
- Some spelling :)

### CODE REVIEW

**Comment by eskombro** [comment]
@react-learner this PR is ready, and even if it shouldn't be merged for now (since we are waiting for the DO review process) you can review it if you feel like it :)

File: resources/howtos/digitalocean_droplet.md [hunk 1]
```diff
@@ -73,44 +73,60 @@ Paste it in your browser. If this screen is shown, your MeiliSearch is now ready
...
+< HTTP/1.1 204 No Content
+...
+```
 
 ![Domain to  MeiliSearch](/digitalocean/11.working-domain.png)
 
 ### 2. Set API KEY and SSL (HTTPS)
 
-Meilisearch is running with a development configuration. It means that you haven't set up an API KEY (anyone can read/write from your MeiliSearch) and you aren't using HTTPS yet. But no worries, the configuration process is automated and very simple. Just connect via SSH to your new MeiliSearch Droplet and answer a few questions:
+Meilisearch is running in a DEVELOPMENT environment. You haven't set up an API KEY (anyone can read/write from your MeiliSearch) and you aren't using HTTPS yet, which makes this configuration uinsafe to une in PRODUCTION. To start the configuration process, connect via SSH to your new MeiliSearch Droplet and follow the instructions:
```
Comment (curquiza) [inline resources/howtos/digitalocean_droplet.md:102]:
```suggestion
Meilisearch is running in a DEVELOPMENT environment. You haven't set up an API KEY (anyone can read/write from your MeiliSearch) and you aren't using HTTPS yet, which makes this configuration unsafe to use in PRODUCTION.
To start the configuration process, connect via SSH to your new MeiliSearch Droplet and follow the instructions:
```

File: resources/howtos/digitalocean_droplet.md [hunk 2]
```diff
@@ -73,44 +73,60 @@ Paste it in your browser. If this screen is shown, your MeiliSearch is now ready
...
 
 ### 2.1. Run the configuration script
 
 Open a terminal and start a new SSH connection with the IP you got from DigitalOcean.
 
 Write in your terminal `ssh root@<your-ip-address>` and press Enter to establish connection:
 
-![Terminal ssh](/digitalocean/12.open-terminal-ssh.png)
+```bash
+ssh meilisearch@42.42.42.42
```
Comment (curquiza) [inline resources/howtos/digitalocean_droplet.md:111]:
```suggestion
$ ssh meilisearch@42.42.42.42
```

File: resources/howtos/digitalocean_droplet.md [hunk 3]
```diff
@@ -73,44 +73,60 @@ Paste it in your browser. If this screen is shown, your MeiliSearch is now ready
...
+```
 
 Write `yes` and press Enter to accept the authentication process.
 
-A script will run automatically, asking for your settings and desired configuration. If you want to run this script again later, you can do so by typing:
+A script will run automatically, asking for your settings and desired configuration. If you want to run this script again anytime, you can do so by using the following command:
 
-`sh /var/opt/meilisearch/scripts/first-login/000-set-meili-env.sh`
+```bash
+meilisearch-setup
```
Comment (curquiza) [inline resources/howtos/digitalocean_droplet.md:119]:
```suggestion
$ meilisearch-setup
```

**Review by curquiza — CHANGES_REQUESTED** [review]
I like these explanations!

**Review by curquiza — APPROVED** [review]

**Review by react-learner — CHANGES_REQUESTED** [review]
LGreatTM 😄 would maybe be nice to replace some of the ALL CAPS with **bold** to make it more consistent with the rest of the docs. But we can always do that later 😌

**Review by curquiza — APPROVED** [review]

**Review by react-learner — APPROVED** [review]

**Pull request merged**